### PR TITLE
refactor(path) Update path for leaflet 1.x compatibility

### PR DESCRIPTION
**Pull request by elesdoar**
Refactored path directive for leaflet 1.x
- remove multipolyline and multipolygon types
- update examples
- update tests
- change bower leaflet dependency and resolution version to 1.0.x

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -14,12 +14,12 @@
     "dist/angular-leaflet-directive.js"
   ],
   "resolutions": {
-    "leaflet": "http://cdn.leafletjs.com/downloads/leaflet-1.0.0-b1.zip"
+    "leaflet": "1.0.x"
   },
   "dependencies": {
     "angular": "1.x",
     "angular-simple-logger": "~0.1.4",
```
Comment (nmccready) [inline bower.json]: TY!

File: doc/paths-attribute.md [hunk 2]
```diff
@@ -25,9 +25,7 @@ Types of paths
 We can define these types of paths:
 
 * polyline.
-* multiPolyline.
```
Comment (nmccready) [inline doc/paths-attribute.md]: Can you point to the new docs on this just for reference?

Comment (nmccready) [inline doc/paths-attribute.md]: Just here in the PR not in the md.

**Comment by nmccready** [comment]
LGTM, just jot down any useful links to leaflet code or docs on 1.X for them.

**Comment by elesdoar** [comment]
OK. 

For polyline
http://mourner.github.io/Leaflet/reference.html#polyline

For polygon
http://mourner.github.io/Leaflet/reference.html#polygon

**Comment by elesdoar** [comment]
Check #851

**Pull request merged**